lib/getHashDigest.ts

```diff
@@ -26,8 +37,12 @@ function divmod32(uint32Array, divisor) {
   return carry;
 }
 
-function encodeBufferToBase(buffer, base, length) {
-  const encodeTable = baseEncodeTables[base];
+function encodeBufferToBase(
+  buffer: Buffer,
+  base: BaseEncodings | number,
```

Comment:
https://github.com/TheLarkInn/loader-utils/blob/f2db90e2a2cbe20f6ccc8f3f313d90a2eff49f25/lib/getHashDigest.ts#L145-L146 because of this can't it be any number?

Reply:
You can cast that to `BaseEncodings`.